src/gl/metal/vcRenderShaders.cpp

```diff
@@ -29,7 +29,7 @@ const char* const g_WaterVertexShader = "waterVertexShader";
 const char* const g_PolygonP3N3UV2FragmentShader = "polygonP3N3UV2FragmentShader";
 const char* const g_PolygonP3N3UV2VertexShader = "polygonP3N3UV2VertexShader";
 
-const char* const g_FlatColour_FragmentShader = "flatColourFragmentShader";
+const char* const g_FlatColour_FragmentShader = "flatColorFragmentShader";
```

Comment:
We usually use Australian English for variable names, any reason for this change?

Reply:
Well no reason specifically that I chose american english, but the change was just for uniformity